packages/react-components/react-breadcrumb/src/components/BreadcrumbButton/BreadcrumbButton.types.ts

```diff
@@ -0,0 +1,27 @@
+import type { ComponentProps, ComponentState } from '@fluentui/react-utilities';
+import { ButtonProps, ButtonSlots, ButtonState } from '@fluentui/react-button';
+import { BreadcrumbProps } from '../Breadcrumb/Breadcrumb.types';
+
+export type BreadcrumbButtonSlots = ButtonSlots;
+
+/**
+ * BreadcrumbButton Props
+ */
+export type BreadcrumbButtonProps = ComponentProps<BreadcrumbButtonSlots> &
+  Pick<BreadcrumbProps, 'appearance' | 'iconPosition' | 'size'> &
+  Pick<ButtonProps, 'disabled'> & {
+    /**
+     * Defines current sate of BreadcrumbButton.
+     *
+     * @default false
+     */
+    current?: boolean;
+  };
+
+/**
+ * State used in rendering BreadcrumbButton
+ */
+export type BreadcrumbButtonState = ComponentState<BreadcrumbButtonSlots> &
+  Omit<ButtonState, keyof ButtonSlots | 'components'> &
+  Required<Pick<BreadcrumbButtonProps, 'current' | 'size'>> &
+  Required<Pick<ButtonProps, 'appearance'>>;
```

Comment:
Is `Required<Pick<ButtonProps, 'appearance'>>` needed? `ButtonState` should contain it already, no?

Reply:
Yes, you're right. Removed it